content/docs/aws/authn-authz.md

```diff
@@ -0,0 +1,116 @@
+
+In Kubeflow 0.6 release, community already move from [Ambassador](https://www.getambassador.io/) to [Istio](https://istio.io/) to manage internal traffic. In AWS solution, TLS, authentication can be done at the ALB and and authorization can be done at Istio layer.
+
+## Enable TLS and Authentication
+
+Right now, certificates for ALB public DNS names are not supported. Instead, you must  prepare a custom domain. You can register your domain in Route53 or any domain provider such as [GoDaddy.com](https://www.godaddy.com/).
+
+[AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) is a service that lets you easily provision, manage, and deploy public and private Secure Sockets Layer/Transport Layer Security (SSL/TLS) certificates for use with AWS services and your internal connected resources.
+
+To get TLS support from the ALB Ingress Controller, you need to follow [this tutorial](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html) to request a certificate in AWS Certificate Manager. After successful validation, you will get a `certificate ARN` to use with the ALB Ingress Controller.
+
+> Note: Even you need to create a `certificate ARN`, we don't necessarily need a custom domain unless you want to use it, you can still use ALB ingress hostname to visit kubeflow central dashboard.
+
+<img src="/docs/images/aws/cognito-certarn.png"
+  alt="Cognito Certificate ARN"
+  class="mt-3 mb-3 border border-info rounded">
+
+[AWS Cognito](https://aws.amazon.com/cognito/) lets you add user sign-up, sign-in, and access control to your web and mobile apps quickly and easily. Amazon Cognito scales to millions of users and supports sign-in with social identity providers, such as Facebook, Google, and Amazon, and enterprise identity providers via SAML 2.0.
+
+<img src="/docs/images/aws/cognito-alb-domain.jpg"
+  alt="Cognito Application Client Setting"
+  class="mt-3 mb-3 border border-info rounded">
+
+For the callback URLs field, if you don't want to use custom domain, you can use ALB hostname for testing. You will see some `Not Secure` warning in browser because ALB hostname doesn't match with domain name in your ACM certificate. It's still functionally working. We will use ALB and please check [section](/docs/aws/custom-domain) to setup custom domain.
+
+<img src="/docs/images/aws/cognito-domain.png"
+  alt="Cognito Domain Name"
+  class="mt-3 mb-3 border border-info rounded">
+
+In order to authenticate and manage users for Kubeflow, let's create a user pool. You can follow these instructions here. Once a user pool created, we will have a `UserPoolId`, a Cognito Domain name, and a Cognito Pool Arn.
+
+Before you `apply all -V`, please update Cognito spec in your Kubeflow
```

Comment:
We don't use `apply all -V` any more. I think you can just remove this bit and just tell them to update the Cognito spec.

Reply:
Change to `kfctl apply -V -f ${CONFIG_FILE}`